packages/react-components/react-checkbox/src/components/Checkbox/useCheckboxStyles.ts

```diff
@@ -196,13 +195,22 @@ const useLabelStyles = makeStyles({
     color: 'inherit',
   },
 
+  before: {
+    marginRight: spacingHorizontalM,
```

Comment:
You can use `tokens.spacingHorizontalM` for this now.

Reply:
👍 will do in another PR

Reply:
* #22982